README.md

```diff
@@ -9,6 +9,40 @@
 npm install screwdriver-artifact-bookend
 ```
 
+## Functionality
+
+Store all the artifacts saved in the Artifacts Directory to the build folder in the Store, to make sure they're not all erased when the build is complete. This bookend currently acts only as a teardown plugin. All artifacts files/folders are written to Store on teardown.
+
+This is a default plugin that is set in the default.yaml (https://github.com/screwdriver-cd/screwdriver/blob/master/config/default.yaml) under the bookends section.
+
+For example, if `SD_ARTIFACTS_DIR=/sd/workspace/artifacts`
+
+And the user creates within that directory:
+
+```
+/sd/workspace/artifacts
+--first_dir
+----first_file.txt
+----second_file.txt
+--second_dir
+----third_file.txt
+----fourth_file.txt
+--fifth_file.txt
+```
+
+Then in Store:
+
+```
+/logs.screwdriver.cd/builds/<BUILD_ID>-ARTIFACTS
```

Comment:
This should be started with `https://{store-uri}/`
where store-uri is what's defined in: https://github.com/screwdriver-cd/screwdriver/blob/master/config/default.yaml#L148